notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>this is amazing, probably it would be interesting to show the rest of the metrics. Also, we can discuss why are the business benefits of optimizing over precision@k and that the rest of the metrics won't be optimal</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/687/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Yes, I can add these. I could also add the times but I need to dig them out of the logs, since there is no API for that.

Reply:
You can simply add end_time - start_time as a metric, right?